app/bingey-ui/src/components/Watchlist/TitleCard/TitleCard.jsx

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react';
+import Card from '@material-ui/core/Card';
```

Comment:
I think using named imports would tidy things up a bit 
```suggestion
import { Card, CardActionArea, ... }from '@material-ui/core';
```
where ... are other components from mui/core